test/install.mk

```diff
@@ -199,3 +199,20 @@ install-prometheus-operator-config:
 	kubectl label ns ${ISTIO_CONTROL_NS} istio-injection=disabled --overwrite
 	# NOTE: we don't use iop to install, as it defaults to `--prune`, which is incompatible with the prom operator (it prunes the stateful set)
 	bin/iop ${ISTIO_CONTROL_NS} istio-prometheus-operator ${BASE}/istio-telemetry/prometheus-operator/ -t ${PROM_OPTS} | kubectl apply -n ${ISTIO_CONTROL_NS} -f -
+
+# This target should only be used in situations in which the kiali operator has not already been installed in a cluster.
+install-kiali-operator: PROM_OP_NS="kiali-operator"
+install-kiali-operator:
+	# make operator-create depends on envsubst command
```

Comment:
We can't require run as root - super bad practice. 
Why is it needed ? And doesn't kiali operator provide any downloadable charts ? 
How about get the stuff you need and check it in istio/installer until they have downloadable stuff - we can maybe wrap with a kustomize command so it's simple to install. 

Keep in mind users will have to follow the same steps.

Reply:
FYI: Kiali Operator is a "black box" and has no consumable internals like downloadable charts for Kiali server. Internally, the operator is implemented as an [Ansible Operator](https://github.com/operator-framework/operator-sdk/blob/master/doc/ansible/user-guide.md), and has an Ansible playbook under the covers which deploys all the yaml required for Kiali server's Deployment, Service, ServiceAccount, Roles, etc. However, the operator's own yaml required to deploy the operator itself are [here](https://github.com/kiali/kiali/tree/master/operator/deploy).

As for requiring something to run as root - I don't know what in the istio-installer requires that, but the Kiali Operator does not need anything like that.

I see that this install.mk file git clones the Kiali repo and just uses `make operator-create`. FWIW, there is no real need to clone the entire thing. You can run the deploy script (downloadable from github or use the shortcut git.io link) and it does the same thing (in fact, operator-create Makefile is nothing more than a call out to that script). That does, however, mean you will pull each of the operator's yaml files from github (rather than locally from the git clone). But in case that is something you want to use instead of git cloning, see https://www.kiali.io/documentation/getting-started for working with the deploy script. It is this: https://raw.githubusercontent.com/kiali/kiali/master/operator/deploy/deploy-kiali-operator.sh